src/main/java/de/tum/cit/aet/artemis/assessment/service/AssessmentService.java

```diff
@@ -283,6 +287,9 @@ public Result saveManualAssessment(final Submission submission, final List<Feedb
         User user = userRepository.getUser();
         result.setAssessor(user);
 
+        // long feedback text is deleted as it otherwise causes duplicate entries errors and will be saved again with resultRepository.save(result)
```

Comment:
I think it would be better to use javadoc here `{@link resultRepository.save}` to ensure that the comment is updated if something is refactored / renamed

Reply:
Is this the right place for the fix then? It feels a little awkward that something has to be deleted for save to work properly - this is not how I would expect save to work 😅

Reply:
the deletion is also weird for me but saving results does not work otherwise as resultRepository.save for some reason also saves the longfeedbacktext in the longfeedbacktextrepository. And because we only override the latest result with a manual correction i do not really see a issue deleting the existing longfeedback from that seperate repository